reactful-numbers/src/app/components/ButtonsBox.js

```diff
@@ -1,67 +1,168 @@
 import React from "react";
 import Button from "./Button";
-function ButtonsBox() {
+
+function ButtonsBox({
+  setInputValue,
+  clearInput,
+  invertSign,
+  percentage,
+  expressionBoxValue,
+  equals,
+}) {
   return (
     <div className="buttons-box">
       <div className="grid-container">
         <div>
-          <Button buttonColor="grey" buttonText="AC"></Button>
+          <Button
+            buttonColor="grey"
+            buttonText={expressionBoxValue == "" ? "AC" : "C"}
```

Comment:
Depending on the state of the expressionBox the buttonText of this button renders differently